libraries/textcomposer/impl/src/main/kotlin/io/element/android/libraries/textcomposer/TextComposer.kt

```diff
@@ -156,17 +156,13 @@ fun TextComposer(
                 targetValue = roundedCornerSize,
                 animationSpec = tween(
                     durationMillis = 100,
-                )
+                ),
+                label = "roundedCornerSizeAnimation"
             )
             val roundedCorners = RoundedCornerShape(roundedCornerSizeState.value)
             val colors = ElementTheme.colors
             val bgColor = colors.bgSubtleSecondary
-
-            val borderColor by remember(state.hasFocus, colors) {
-                derivedStateOf {
-                    if (state.hasFocus) colors.borderDisabled else bgColor
-                }
-            }
+            val borderColor = colors.borderDisabled
```

Comment:
Double checking: are we simplifying how the border color is applied?

Reply:
Yeah this is intentional - the border is now always visible regardless of the state